contractdata/PARIS/ESCALATION/XMAS/XMAS3.json

```diff
@@ -21,16 +21,47 @@
                     "States": {
                         "Start": {
                             "Disguise": {
+                                "Condition": {
+                                    "$or": [
+                                        {
+                                            "$eq": [
+                                                "$Value",
+                                                "315400cd-90d8-43cc-8c22-62c0cb8969a5"
+                                            ]
+                                        },
+                                        {
+                                            "$eq": [
+                                                "$Value",
+                                                "31ac5259-ff59-46e0-ab0f-8ddeaf296a36"
+                                            ]
+                                        }
+                                    ]
+                                },
+                                "Transition": "Success"
+                            },
+                            "StartingSuit": {
                                 "Condition": {
                                     "$eq": [
                                         "$Value",
-                                        "315400cd-90d8-43cc-8c22-62c0cb8969a5"
+                                        "31ac5259-ff59-46e0-ab0f-8ddeaf296a36"
                                     ]
                                 },
                                 "Transition": "Success"
                             }
                         },
-                        "Success": { "Disguise": { "Transition": "Failure" } }
+                        "Success": {
+                            "Disguise": {
```

Comment:
```suggestion
                            "Disguise": {
                                "Condition": {
                                    "$not": {
                                        "$or": [
                                            {
                                                "$eq": [
                                                    "$Value",
                                                    "315400cd-90d8-43cc-8c22-62c0cb8969a5"
                                                ]
                                            },
                                            {
                                                "$eq": [
                                                    "$Value",
                                                    "31ac5259-ff59-46e0-ab0f-8ddeaf296a36"
                                                ]
                                            }
                                        ]
                                    }
                                },
```

Reply:
Honestly speaking, it's not like I haven't tried this before, but whenever I write it this way, my game can't read the contract (I'm guessing it might be that the `$not` range is too large). I was testing it by converting it to a user contract.

Reply:
`$not` accepts 1 child, not multiple